src/main/platform.h

```diff
@@ -31,6 +31,7 @@
 #include "stm32f7xx_hal.h"
 #include "system_stm32f7xx.h"
 
+#include "stm32f7xx_ll_cortex.h"
```

Comment:
Is this needed here?

Reply:
It's the place where all LL/HAL headers are included in our project, so it belongs here for consistency